### ISSUE TITLE
Flex depended on to build, but not listed as build dependency in debian/control, nor checked in ./configure

### ISSUE THREAD

**Issue by zafnz** [wandnz/amplet2#7]
flex is required but not depended on in debian build dependencies, nor is it's presence checked for in configure script.

```
.....snip......
make[5]: Entering directory '/home/zaf/dev/amplet2/src/tests/http'
Making all in .
make[6]: Entering directory '/home/zaf/dev/amplet2/src/tests/http'
  CC       http.lo
  CC       servers.lo
  CC       parsers.lo
  CC       output.lo
  LEX      lexer.c
/home/zaf/dev/amplet2/missing: line 81: flex: command not found
WARNING: 'flex' is missing on your system.
         You should only need it if you modified a '.l' file.
         You may want to install the Fast Lexical Analyzer package:
         <http://flex.sourceforge.net/>
Makefile:582: recipe for target 'lexer.c' failed
make[6]: *** [lexer.c] Error 127
make[6]: Leaving directory '/home/zaf/dev/amplet2/src/tests/http'
```

Will submit pull request for this shortly

**Comment by zafnz** [comment]
This is resolved by the above pull request

**Comment by brendonj** [comment]
Merged #8. My build environment must not be as clean as I thought it was.
